docs/install/pubsub-service-account.md

```diff
@@ -56,27 +56,75 @@ its improved security properties and manageability. For more information about
 Workload Identity see
 [here](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity).
 
-**_Note:_** If you install the Knative-GCP Constructs with v0.14.0 or older
+**_Note:_** 
+1. If you install the Knative-GCP Constructs with v0.14.0 or older
 release, please use option 2.  
-**_Note:_** `spec.googleServiceAccount` in v0.14.0 is deprecated for security
+2. `spec.googleServiceAccount` in v0.14.0 is deprecated for security
 implications. It has not been promoted to v1beta1 and is expected to be removed
 from v1alpha1 in the v0.16.0 release. Instead, `spec.serviceAccountName` has
 been introduced for Workload Identity in v0.15.0, whose value is a Kubernetes
 Service Account.
 
-Using the Google Cloud Service Account you just created and checking
+There are two scenarios to leverage Workload Identity for resources in the Data Plane:
+* ***Non-default scenario:***
+
+    Using the Google Cloud Service Account `cre-pubsub` you just created and checking
 [Option 1 (Recommended): Workload Identity](../install/authentication-mechanisms-gcp.md/#option-1-recommended-workload-identity)
 in
-[Manually Configure Authentication Mechanism for GCP](../install/authentication-mechanisms-gcp.md)
+[Authentication Mechanism for GCP](../install/authentication-mechanisms-gcp.md)
 to configure Workload Identity in the namespace your resources will reside.
 
-You will have a Kubernetes Service Account after the above configuration, which
-is bound to the Google Cloud Service Account you just created. Remember to put
+    You will have a Kubernetes Service Account after the above configuration, which
+is bound to the Google Cloud Service Account `cre-pubsub`. Remember to put
 this Kubernetes Service Account name under `spec.serviceAccountName` when you
 follow
 [example](https://github.com/google/knative-gcp/tree/master/docs/examples) to
 use resource.
-
+* ***Default scenario:***
+    
+    Instead of manually configuring Workload Identity with [Authentication Mechanism for GCP](../install/authentication-mechanisms-gcp.md), 
```

Comment:
```suggestion
Instead of manually configuring Workload Identity with [Authentication Mechanism for GCP](../install/authentication-mechanisms-gcp.md), 
```